src/ucp/wireup/select.c

```diff
@@ -430,9 +444,10 @@ ucp_wireup_select_transport(ucp_ep_h ep, const ucp_address_entry_t *address_list
         *(p - 2) = '\0'; /* trim last "," */
     }
 
-    select_info->rsc_index  = best_rsc_index;
-    select_info->addr_index = best_dst_addr_index;
-    select_info->score      = best_score;
+    select_info->rsc_index   = best_rsc_index;
+    select_info->addr_index  = best_dst_addr_index;
+    select_info->score       = best_score;
+    select_info->max_num_eps = best_max_num_eps;
```

Comment:
why do we need to return max_num_eps ??
we can tell it from rsc_index

Reply:
removed, we don't need it